src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // делим дату на части
+  const dateParts = date.split(fromFormat[3]);
```

Comment:
You are using `fromFormat[3]` directly to split the `date`. It would be more clear if you used a descriptive variable name like `const delimiter = fromFormat[3];` for better code readability.